src/contexts/authContext.js

```diff
@@ -40,12 +40,14 @@ const AuthProvider = props => {
     }
   }
   const login = form => AuthService.login(form).then(reload);
+  const saveLinkedInToken = token =>
+    AuthService.saveLinkedInToken(token).then(reload);
```

Comment:
Can we just use `TokenUtils.storeLinkedInToken` instead? Since AuthService is literally just a wrapper for that. Delete `AuthService.saveLinkedInToken`